client/verta/tests/unit_tests/internal_utils/test_kafka_utils.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+
+"""Unit tests for Kafka utilities."""
+
+from typing import Any, Dict, List
+
+import pytest
+
+from verta._internal_utils import kafka
+
+
+@pytest.fixture
+def mock_kafka_configs_response() -> Dict[str, Any]:
+    """
+    Provide mocked API result from `api/v1/uac-proxy/system_admin/listKafkaConfiguration`
+    with a single Kafka configuration.
+    """
+    return {
+        "configurations": [
+            {
+                "id": "1232abcdefg4-567h-891i-0jkl-1m1n1o2131pq",
+                "kerberos": {
+                    "enabled": True,
+                    "conf": (
+                        "[libdefaults]\ndefault_realm ="
+                        " EXAMPLE.COM\n\n[realms]\nEXAMPLE.COM"
+                        " = {\n    kdc_ports = 12,345\n    kadmind_port = 123\n    kdc ="
+                        " testing--kafka-server-X.integrations--kafka-server-headless.ci.svc.cluster.local\n"
+                        "    admin_server ="
+                        " testing--kafka-server-X.integrations--kafka-server-headless.ci.svc.cluster.local\n}"
+                    ),
+                    "keytab": "SLG45KJlksajdfglakjWlskadfj098lkjakdfjl1k3kjflakdj-lakf+alkqfvmLPWZNVTdfS84UHads3dlkfj97alfkj!lakdfjlkaj8fg=",
+                    "client_name": "test-kafka-client_name",
+                    "service_name": "test-kafka-utils",
+                },
+                "brokerAddresses": "integrations--kafka-X.integrations--kafka-headless.ci.svc.cluster.local:00001",
+                "enabled": True,
+                "name": "Test Kafka Name",
+            }
+        ]
+    }
+
+
+@pytest.fixture
+def mock_kafka_topics() -> List[str]:
+    """Provide a mock API response from `api/v1/deployment/list/kafka-topics`."""
+    return ["test-topic-1", "test-topic-2", "test-topic-3"]
```

Comment:
If you'd like (and I'd mildly recommend it), these two fixtures would be straightforward to use `hypothesis` for.

Reply:
I'm going to try to tackle this in another PR.